addon-test-support/run-options.ts

```diff
@@ -0,0 +1,25 @@
+import { RunOptions } from 'axe-core';
+import { getContext } from '@ember/test-helpers';
+import { registerDestructor } from '@ember/destroyable';
+
+let optionsStack: RunOptions[] = [];
+
+/**
+ * Sets run options specific to a test.
+ *
+ * @param options Axe {RunOptions} to be provided to the audit helper.
+ */
+export function setRunOptions(options: RunOptions = {}) {
+  optionsStack.push(options);
+
+  registerDestructor((getContext() as any).owner, () => optionsStack.pop());
```

Comment:
I'm not 100% that this is correct. I'm not sure that attaching the destructor to the owner is enough to correctly attribute the options to all levels of options (global, test module level, test level). Would the global invocation even have an owner?